webhooks.md

```diff
@@ -47,6 +47,31 @@ It is furthermore recommended to add support for running the operator locally wi
 
 When the associated operator is deployed via OLM, its webhooks are installed in the cluster and cert management is automatically handled as well.  You don't need to do anything differently than you normally would for installing the operator.
 
+### Changing webhook defaults in OLM context
+
+To change the webhook environment variable defaults (those mentioned [here](https://github.com/openstack-k8s-operators/docs/blob/main/webhooks.md#why-did-we-introduce-webhooks)) for an operator that was deployed via OLM, do the following _after_ you have installed the operator and its CSV reaches the `Succeeded` state:
+
+1. Create a patch file to change the desired default
+
+```bash
+oc get csv -n openstack-operators -l operators.coreos.com/<OPERATOR_NAME>.openstack-operators -o=jsonpath='{.items[0]}' | jq '(.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[] | select(.name=="<DEFAULT NAME>")) |= (.value="<NEW_DEFAULT_VALUE>")' > default_patch.out
```

Comment:
Would the manager be the second container always?

Reply:
As far as I have ever seen, yes.  The first is always seems to be the `kube-rbac-proxy`.